local/src/test/java/com/kdjj/local/dataSource/RecipeImageLocalDataSourceImplTest.kt

```diff
@@ -2,56 +2,133 @@ package com.kdjj.local.dataSource
 
 import com.kdjj.local.ImageFileHelper
 import com.kdjj.local.dao.UselessImageDao
+import com.kdjj.local.dto.UselessImageDto
 import kotlinx.coroutines.runBlocking
-import org.junit.Assert.assertEquals
+import org.junit.Assert.*
 import org.junit.Before
 import org.junit.Test
-import org.mockito.Mockito.`when`
-import org.mockito.Mockito.mock
-
-//class RecipeImageLocalDataSourceImplTest {
-//
-//	private lateinit var mockImageFileHelper: ImageFileHelper
-//	private lateinit var mockUselessImageDao: UselessImageDao
-//	private lateinit var recipeImageLocalDataSourceImpl: RecipeImageLocalDataSourceImpl
-//	private val testUri = "this is test uri"
-//	private val testByteArray = testUri.toByteArray()
-//	private val testDegree = 90f
-//	private val testPair = Pair(testByteArray, testDegree)
-//
-//	@Before
-//	fun setup() {
-//		mockUselessImageDao = mock(UselessImageDao::class.java)
-//		mockImageFileHelper = mock(ImageFileHelper::class.java)
-//		recipeImageLocalDataSourceImpl = RecipeImageLocalDataSourceImpl(mockImageFileHelper, mockUselessImageDao)
-//	}
-//
-//	@Test
-//	fun convertToByteArray_getImageByteArrayAndImageDegree_true(): Unit = runBlocking {
-//		//given
-//		`when`(recipeImageLocalDataSourceImpl.convertToByteArray(testUri)).thenReturn(
-//			Result.success(testByteArray to testDegree)
-//		)
-//		//when
-//		val result = recipeImageLocalDataSourceImpl.convertToByteArray(testUri)
-//		//then
-//		assertEquals(testPair, result.getOrNull())
-//	}
-//
-//	@Test
-//	fun convertToInternalStorageUri_localStorageImageUri_true(): Unit = runBlocking {
-//		//given
-//		`when`(
-//			recipeImageLocalDataSourceImpl.convertToInternalStorageUri(
-//				testByteArray,
-//				"fileName",
-//				testDegree
-//			)
-//		).thenReturn(Result.success(testUri))
-//		//when
-//		val newLocalImagePathResult =
-//			recipeImageLocalDataSourceImpl.convertToInternalStorageUri(testByteArray, "fileName", testDegree)
-//		//then
-//		assertEquals(testUri, newLocalImagePathResult.getOrNull())
-//	}
-//}
+import org.mockito.Mockito.*
+
+class RecipeImageLocalDataSourceImplTest {
+
+    private lateinit var mockImageFileHelper: ImageFileHelper
+    private lateinit var mockUselessImageDao: UselessImageDao
+    private lateinit var recipeImageLocalDataSourceImpl: RecipeImageLocalDataSourceImpl
+
+    private val dummyUri = "this is test uri1"
+
+    private val dummyUriList = listOf(
+        "this is test uri1",
+        "this is test uri2",
+        "this is test uri3",
+        "this is test uri4"
+    )
+
+    private val dummyByteArrayWithDegreeList = listOf(
+        Pair("this is test uri1".toByteArray(), 90f),
+        Pair("this is test uri2".toByteArray(), 90f),
+        Pair("this is test uri3".toByteArray(), 90f),
+        Pair("this is test uri4".toByteArray(), 90f)
+    )
+
```

Comment:
아래있는 리스트들을 합친 것과 내용이 같은것 같은데 zip을 이용하면 어떨까요?
```suggestion
    private val dummyByteArrayWithDegreeList = dummyByteArrayList.zip(dummyDegreeList)

```